packages/nhost-js/src/core/nhost-client.ts

```diff
@@ -27,12 +17,11 @@ export class NhostClient {
    * Nhost Client
    *
    * @example
-   * const nhost = new NhostClient({ url });
+   * const nhost = new NhostClient({ subdomain, region });
```

Comment:
Could you please wrap this in backticks and put `ts` to make it properly styled in the docs?